easybuild/easyconfigs/l/libpng/libpng-1.6.16-intel-2015a.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'ConfigureMake'
+
+name = 'libpng'
+version = '1.6.16'
+
+homepage = 'http://www.libpng.org/pub/png/libpng.html'
+description = "libpng is the official PNG reference library"
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
```

Comment:
I will look into that at some point, but not without a script that allows us to enforce this consistently...